eopdev/ap-southeast-2/eopdev/services/ecs-eop-manager/module_config.hcl

```diff
@@ -1,5 +1,5 @@
 # Define some config vars that can be imported by the shared terragrunt config. To keep the config dry.
 locals {
   config_secrets_manager_arn = "arn:aws:secretsmanager:ap-southeast-2:657968434173:secret:EOPManagerConfig-cWXx3Q"
-  container_image_tag        = "b0023ecbf90b99078afef607ac6ca0d752f64c14"
+  container_image_tag        = "9cd31e470fa1b8576f9f22d8127d5fced42cee83"
```

Comment:
This needs updating and re-deploying once we've merged the [eop app PR.](https://github.com/Greater-Wellington-Regional-Council/Environmental-Outcomes-Platform/pull/23)